kubernetes/istio_details_service.go

```diff
@@ -45,6 +46,15 @@ func (in *IstioClient) GetIstioDetails(namespace string, serviceName string) (*I
 	return &istioDetails, nil
 }
 
+func (in *IstioClient) DeleteIstioObject(namespace string, objectType string, objectName string) (error) {
+	log.Infof("DeleteIstioObject input: %s %s %s", namespace, objectType, objectName)
+	_, err := in.istioNetworkingApi.Delete().Namespace(namespace).Resource(objectType).Name(objectName).Do().Get()
```

Comment:
That looks weird: `.delete() ... .get()`, really? :)

Reply:
I guess I expected the final Get() to retrieve the return code.

If you look around, you'll also see `Api.Get().Namespace(namespace).Resource(virtualServices).Do().Get()` where there are also two `Get()` :-)